api-reference/media-api/rename-file.md

```diff
@@ -127,4 +127,28 @@ imagekitio.rename_file(
 )
 ```
 {% endtab %}
+{% tab title="PHP" %}
+```php
+use ImageKit\ImageKit;
+
+$public_key = "your_public_api_key";
+$your_private_key = "your_private_api_key";
+$url_end_point = "https://ik.imagekit.io/your_imagekit_id";
+
+$imageKit = new ImageKit(
+    $public_key,
+    $your_private_key,
+    $url_end_point
+);
+
+$filePath='/sample-folder/sample-file.jpg';
+$newFileName = 'sample-file2.jpg';
+$renameFile = $imageKit->rename([
+    'filePath' => $filePath,
+    'newFileName' => $newFileName,
+], false);
```

Comment:
Maybe take this `false` into a variable so that the user can understand what it means?